docs/admin/install.rst

```diff
@@ -336,9 +336,77 @@ This command will take a considerable amount of time and approximately 4GB of ba
+
+After confirming this change, you can launch KeePassXC from the **Domain: vault** menu. On first start, it will suggest enabling automatic updates, which you should decline: ``vault`` is networkless, so the built-in update check will fail; the app will be updated through system updates instead.
+
+When creating a new password database, KeePassXC will prompt you to protect it with a password, and will refuse to accept a blank password. If you want to configure easy access to the password database, you can protect the database using a key file, via **Add additional protection > Add Key File > Generate** on the final screen. Once you have configured a key file, KeePassXC will accept leaving the master password field blank.
+
+Clipboard configuration
+~~~~~~~~~~~~~~~~~~~~~~~
+Qubes supports copying the clipboard contents from one VM to another using `special keyboard shortcuts <https://www.qubes-os.org/doc/copy-paste/>`_. It also permits configuring the access policies for each VM's clipboard. By default, cross-VM clipboard copying is completely disabled for all VMs created as part of the installation (the ones that start with ``sd-``) for security reasons.
+
+..  important::
+
+    Before changing the clipboard configuration, we recommend reviewing the section :doc:`Managing Clipboard Access <managing_clipboard>` of this guide, which goes into further detail on the security considerations.
+
+If you did set up a password manager in the previous step, you may want to enable copying passwords from KeePassXC to the SecureDrop Client. The password manager runs in the networkless ``vault`` VM, and the SecureDrop Client runs in the ``sd-app`` VM. To permit this one-directional clipboard use, issue the following command in ``dom0``:
 
-Once the update check is complete, the client will launch. Log in using an existing journalist account and verify that sources are listed and submissions can be downloaded, decrypted, and viewed.
+.. code-block:: sh
+
+  qvm-tags vault add sd-send-app-clipboard
+
+To remove the permission, use:
+
+.. code-block:: sh
+
+  qvm-tags vault del sd-send-app-clipboard
+
+These policy changes take effect immediately.
+
+Reviewing and exporting logs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+SecureDrop Workstation aggregates system logs from all its VMs in the ``sd-log`` VM, in the folder ``~/QubesIncomingLogs``, with one subfolder for each VM. While details about submissions are not logged, the logs may contain sensitive information, such as timing data, the two-word designation for a given source, or revelatory errors. For this reason, the ``sd-log`` VM is networkless, and you cannot copy files from ``sd-log`` to other VMs by default.
+
+If you want to selectively enable copying logs to a target VM such as ``work``, you can use a command like the following in ``dom0``:
```

Comment:
There should be also be a disclaimer here about the risks of sharing these logs with other VMs, specifically that logs contain Submission and reply metadata, as well as usage information pertaining to the SecureDrop workstation and servers that is not public. It might make more sense to have all these commands consolidated later, where the risks are clearly defined.

Reply:
It currently says:

> the logs may contain sensitive information, such as timing data, the two-word designation for a given source, or revelatory errors.

I can spell that out in a bit more detail as a warning.

Reply:
Done in 5d221f0eca63259c01d8583c059930667ea0bddd. I've split this into its own admin section in this commit; in spite of the command similarities with clipboard access, my bias is to keep it in a separate section, since we may want to expand each section in different ways (e.g., we may want to flesh out the logs section in future to give some more pointers about how to redact logs, what to look for, etc.).